Changelogs for Digital Twin Manager UI 1021.1.2 (#3105)
SoftwareAG/c8y-docs: content/change-logs/application-enablement/dtm-1021-2-1-update-sentence-for-child-asset-models-in-the-new-asset-type-screen.md

---
date: ""

> title: "Update sentence for child asset models in the New Asset Type screen"

Reviewer: ```suggestion
title: "Improved info message when no Child Asset Models are selected when creating or updating Asset Models"
```

> title: "Improved info message when no Child Asset Models are selected when creating or updating Asset Models"

Reviewer 2: ```suggestion
title: "Improved info message when no child asset models are selected"
```

product_area: "Application enablement & solutions"
change_type:
    - value: "change-VSkj2iV9m"
      label: "Fix"
component:
    - value: "component-Tl88RYb4A"
      label: "Digital Twin Manager"
build_artifact:
    - value: "tc-wYIY0MBDO"
      label: "dtm"
ticket: "CTM-1051"
version: "1021.2.1"
---

> Previously, if no allowed asset models were selected, the message displayed as "No allowed child assets," which caused confusion as the field describes about allowed child asset models. This has now been updated to "No allowed child asset models selected" for better clarity.

Reviewer: ```suggestion
Previously, if no child asset models were selected while creating or updating an Asset Model, the message displayed as 'No allowed child assets,' which was misleading since the field specifically refers to allowed child asset models. To enhance clarity, the message has been updated to 'No allowed child asset models selected,' ensuring better alignment with the field's purpose and reducing potential confusion.
```

> Previously, if no child asset models were selected while creating or updating an Asset Model, the message displayed as 'No allowed child assets,' which was misleading since the field specifically refers to allowed child asset models. To enhance clarity, the message has been updated to 'No allowed child asset models selected,' ensuring better alignment with the field's purpose and reducing potential confusion.

Reviewer 2: ```suggestion
Previously, if no child asset models were selected while creating or updating an asset model, the message "No allowed child assets" was displayed, which was misleading. To enhance clarity, the message has been updated to "No allowed child asset models selected", ensuring better alignment with the field's purpose and reducing potential confusion.
```
